### ISSUE TITLE
Join the VSE Facebook Group?

### ISSUE THREAD

**Issue by tin2tin** [Botmasher/blender-vse-customizations#1]
Not a bug report, but I noticed that you are working on a complex VSE related script, so I would like to invite you to a new VSE Facebook Group. It is maybe only one week old, but it is very active and with good exchange on add-ons. I'm not the initiator of the group, but there are a lot of good people there with the common goal of improving the VSE. I dislike FB like most people, but the vibe is quite good in this group. Anyway here's a link for the group: [https://www.facebook.com/groups/1981104845289453/](https://www.facebook.com/groups/1981104845289453/) It's called "Video Editing in Blender Sequencer".

**Comment by Botmasher** [comment]
Thank you for the link. I'm not on FB and was unaware of the group, but I'll definitely check out the conversation going on there.

**Comment by tin2tin** [comment]
Great. Maybe you could add links and short descriptions of you VSE add-ons to this document, so people will know about them? [https://docs.google.com/document/d/1K7MM-JpcrRIclhXKHOpBHCpBVLq0KZ0W5BocKaEUu6o/edit?usp=drivesdk](https://docs.google.com/document/d/1K7MM-JpcrRIclhXKHOpBHCpBVLq0KZ0W5BocKaEUu6o/edit?usp=drivesdk)

**Comment by Botmasher** [comment]
Certainly. It looks like you had a chance to add entries for many of these scripts. I'll go through the doc this afternoon and see what I can add.

**Comment by tin2tin** [comment]
That would be great. Please do with it as you prefer. I just felt that there are too many good things you have done here. And it would be sad if people do not find out.

**Comment by Botmasher** [comment]
I see the scripts listed there, so I added some descriptions. I tried to stick to the existing formatting. I'll also be taking a look at some of the other projects I haven't seen before.

**Comment by tin2tin** [comment]
I don't know if you're following all the good things happening with the VSE right now? 

The VSE was the theme for Blender.today - last night: https://www.youtube.com/watch?v=huVy3EP1Pgg
Yesterday Richard released his first WIP patch for improved cache handling(meaning faster playback): https://developer.blender.org/D3934
Now there is both a VSE chat room: https://blender.chat/channel/vse
And a VSE dedicated channel at the blender.community: https://blender.community/c/vse/

Anyway, I've been tinkering with a small script to import storyboards from Storyboarder into Blender and I ran into the problem you fixed with your pretty_img.py, so if I credit you properly, can I use your script?

**Comment by Botmasher** [comment]
Certainly, please do use it. I should pick a license and set it for this project for the future.

I haven't been following closely, so I thank you for the links. Time for me to catch up!
